posix.mak

```diff
@@ -30,6 +31,13 @@ STABLE_DMD_ROOT=/tmp/.stable_dmd-$(STABLE_DMD_VER)
 STABLE_DMD_URL=http://downloads.dlang.org/releases/2014/dmd.$(STABLE_DMD_VER).$(OS).zip
 STABLE_DMD=$(STABLE_DMD_ROOT)/dmd2/$(OS)/$(if $(filter $(OS),osx),bin,bin$(MODEL))/dmd
 
+# exclude lists
+MOD_EXCLUDES_PRERELEASE=$(addprefix --ex=, gc. rt. core.internal. core.stdc.config core.sys.	\
+	std.c. std.algorithm.internal std.internal. std.regex.internal. std.typelist		\
+	std.windows. etc.linux.memoryerror)
+
+MOD_EXCLUDES_RELEASE=$(MOD_EXCLUDES_PRERELEASE) --ex=core.stdc.
```

Comment:
There was some grumbling on iRC today about us not even having a list of available C functions (even with placeholder documentation). It might be worth it to list them if only because you can use a web search to find which module a function is in.